### DISCUSSION TITLE
Move to ubuntu 22.04 and transmission 3.0

### DISCUSSION THREAD

**Discussion by GAS85** [haugene/docker-transmission-openvpn#2337] [Ideas]
Hey,

how about to move to newer LTS 22.04, that will give us transmission daemon 3.0, that is already a bit old. but much newer as current 2.94 in container.
Root cause: there are few issues that are only solved in 3.0, or even will be solved in 4.0 in future...

**Comment by ilike2burnthing** [comment]
https://github.com/haugene/docker-transmission-openvpn/discussions/1937

**Comment by GAS85** [comment]
How about to add https://github.com/aria2/aria2 as alternative downloader client? It has also bunch of UIs supported.

**Reply by pkishino** [reply]
The whole point of this container is a minimal openvpn-transmission package.. if you want a different client your better off creating a separate container
